lib/codecs/Cargo.toml

```diff
@@ -21,7 +21,7 @@ lookup = { package = "vector-lookup", path = "../vector-lookup", default-feature
 memchr = { version = "2", default-features = false }
 once_cell = { version = "1.19", default-features = false }
 ordered-float = { version = "4.2.2", default-features = false }
-prost = { version = "0.12.6", default-features = false, features = ["std"] }
+prost = { version = "0.13.1", default-features = false, features = ["std"] }
```

Comment:
Note: does this need to be a different version? I would make this a workspace dependency and use the same crate version everywhere.

Reply:
Agreed, this seems like a good candidate for a workspace dependency.

Reply:
👍 I can followup on this. I also see a check failing, probably an incompatible change was introduced.

Reply:
Closing the loop: https://github.com/vectordotdev/vector/pull/21426